src/api/v2/index.php

```diff
@@ -317,6 +317,6 @@ public static function addCORSheaders(Request $request, $response) {
 require __DIR__ . "/../../inc/apiv2/helper/setUserPassword.routes.php";
 require __DIR__ . "/../../inc/apiv2/helper/taskExtraDetails.routes.php";
 require __DIR__ . "/../../inc/apiv2/helper/unassignAgent.routes.php";
-
+require __DIR__ . "/../../inc/apiv2/helper/getCracksOfTask.routes.php";
```

Comment:
Ideally, we sort the imports in the same way they are sorted in a directory by default (alphabetically). It's easier to see if all imports are there.